_data/simple-icons.json

```diff
@@ -11114,6 +11114,12 @@
             "hex": "8A9296",
             "source": "https://scrutinizer-ci.com"
         },
+        {
+            "title": "ScyllaDB",
+            "hex": "6CD5E7",
+            "source": "https://raw.githubusercontent.com/Grifed-source/simple-icons/19ba733a0c29af1747ffad59b1016595b453c4c6/icons/scylladb.svg",
```

Comment:
Source still needs updating to the media kit link. We can't use your fork of our repo as the source.